app/controllers/api/v1x0/mixins/rbac_mixin.rb

```diff
@@ -130,19 +130,14 @@ def owner_request_ids
         # All child request ids for approver to process
         def visible_request_ids_for_approver
           visible_states = [ApprovalStates::NOTIFIED_STATE, ApprovalStates::COMPLETED_STATE]
-          Request.where(:workflow_id => workflow_ids, :state => visible_states).pluck(:id)
+          Request.where(:group_ref => assigned_group_refs, :state => visible_states).pluck(:id)
```

Comment:
This looks good, just wanted to comment that `visible_states` should be a constant, either defined here or in approval_states.rb.